src/rpdk/core/boto_helpers.py

```diff
@@ -64,7 +64,7 @@ def get_temporary_credentials(session, key_names=BOTO_CRED_KEYS, role_arn=None):
             creds = (frozen.access_key, frozen.secret_key, frozen.token)
         else:
             try:
-                response = sts_client.get_session_token()
+                response = sts_client.get_session_token(DurationSeconds=900)
```

Comment:
this change looks good. I am curious, how will we handler contract test run for resources which take time to create/update.

Reply:
It actually will re-invoke the handler if the response is IN_PROGRESS: https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/contract/resource_client.py#L506